cms/core/blocks/base.py

```diff
@@ -0,0 +1,82 @@
+from django.core.exceptions import ValidationError
```

Comment:
note: this wasn't necessary per se, in that #41 does it, and we're likely to get a merge conflict, so it generates a bit of extra work. The move makes sense in #41 because of the other block changes